.github/templates/linux_ci_workflow.yml.in

```diff
@@ -178,3 +178,86 @@ jobs:
           docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
           # Prune all of the docker images
           docker system prune -af
+  {% if enable_doc_jobs %}
+  pytorch_python_doc_build:
+    runs-on: ubuntu-18.04
+    needs: calculate-docker-image
```

Comment:
thanks, yes sure, I may need to figure out how to iterate this faster

every change requires approvals - so each iteration requires a few hours of wait - anything special I can do to test the flow e2e on my own before requiring your review time?

Reply:
yeah this is unfortunately a pain point that we haven't yet solved; one hacky solution we could take in the short-term is have you make a tiny cosmetic diff and Jedi-land it so that GitHub no longer thinks you're a "first-time contributor", after which (I hope) you wouldn't need approval to run CI on this PR; @seemethere what do you think?

Reply:
as for running locally, I don't think we have a good solution for that yet either (but I could be wrong)

Reply:
tested a few times, and yes, indeed it requires the artifacts generated from the `build` step, I updated and may need a new approval for the workflows.

Noticeable changes are
- refactor `docker cp` into a single `docker run`
- download artifacts from `build` step and mount as a volume
- change from `sh` to `bash` to run in docker as it requires `pushd` which is a bash-only command
- refactor circleci tag into GITHUB_REF - as gha only supports git ref

Reply:
To pass the "first-time contributor" PR https://github.com/pytorch/pytorch/pull/57779